app/src/main/AndroidManifest.xml

```diff
@@ -2,14 +2,19 @@
 <manifest xmlns:android="http://schemas.android.com/apk/res/android"
     xmlns:tools="http://schemas.android.com/tools">
 
+    <uses-permission android:name="android.permission.INTERNET" />
+    <uses-permission android:name="android.permission.ACCESS_NETWORK_STATE" />
```

Comment:
Why is it required? Firebase works without these permissions.

Reply:
I was gonna test something. Forgot to remove them.